featherbed-core/src/main/tut/02-basic-usage.md

```diff
@@ -30,7 +30,8 @@ Create a `Client`, passing the base URL to the REST endpoints:
 
 ```tut:book
 import java.net.URL
-val client = new featherbed.Client(new URL("http://localhost:8765/api/"))
+import featherbed.Client
```

Comment:
You are right, of course, but since in docs we are talking only about Featherbed's client, fully qualified names look cumbersome.
But anyway it's up to you whether to merge this :)